docs/competition/matches/match-scoring.md

```diff
@@ -11,47 +11,76 @@ Any roles related to scoring matches go here.
 
-As well as scoring, your responsibilities include making sure robots start in the right corner and switching off robots that do not automatically turn off after a match ends or when a team requests that you do so. You must also make sure that any arena props (tokens, for example) are in the correct place at the start of each match as detailed below. You are not responsible, however, for making sure a match starts with the correct number of teams -- that is down to the shepherds.
+Your responsibilities are:
+- Recording the state of the arena at the exact end of the match (see 'Scoring').
+- Ensuring robots start in the correct corner, and are entirely within their starting zone.
+- Ensuring no team members touch a robot after it has first moved. (see 'Teams touching robots').
+- Switching off robots when a team requests that you do so, or if they do not automatically turn off after a match ends.
+- Resetting the arena by moving props (tokens, for example) back to the correct place after you and your partmer have both recorded the end-state of the match. There is no mechanism for delaying the matches, so please request extra help if you are struggling to make the deadlines.
 
-Teams must not be allowed in the arena during a match and only one team member is allowed in the area between the crowd barrier and arena wall. It is the responsibility of the team members to place their robot in the start zone before the match starts. If a robot moves off before the match starts (aside from minor servo-resetting arm movements) teams are allowed to switch off their robot, reset it and start again. However, if the mis-started robot touches any arena prop then they are immediately disqualified. Once the robot start button has been pressed and the match has started, teams are not allowed to interact with their robots until after scoring has taken place.
+You are not responsible for:
+- Making sure a match starts with the correct number of teams -- that is down to the shepherds.
 
-If the team members indicates that they wish for you to turn off their robot (i.e. "throwing in the towel") you may do so but by doing so they forfeit the match which will be recorded in the score sheets as a disqualification. There are two cases where you are allowed to switch off a robot yourself, one is when the match ends and the other is when a robot is causing **unintentional and significant** damage to another robot (i.e. mashes into another robot, pins it, stopping it from being able to function -- includes one dragging the other around if they become intertwined), and in that case it is up to you to decide which robot should be switched off but their match is not forfeited. If the damage is intentional, then the team will get forfeited (potentially for previous matched too), but this decision should be left to the judge. A checklist for robot damage is available below.
+### Summary of the most important rules & regulations for SR2019:
```

Comment:
I'm not sure how useful this section is? We already recommend they read the rules first, which covers all these points. 

This is also the only thing which makes this page year specific, something we should be avoiding as much as possible

Reply:
I'd normally say so, but as Peter said earlier, they likely won't have read the rules, and this role is important enough for them to really need to have read them so I think it's worth putting it in.

I've tried to always say "SR2019" whenever I put year-specific stuff, so it's much easier to find areas which need updating.

Reply:
Works for me :+1: